torch/distributed/rpc/backend_registry.py

```diff
@@ -45,6 +45,7 @@ def process_group_init_backend_handler(
     self_rank,
     worker_name_to_id,
     num_send_recv_threads,
```

Comment:
Can we remove `num_send_recv_threads ` as it's not common to all RPC agents at the moment.

Reply:
This is a function specifically for initializing PG agent though, wouldn't it be better to have the argument explicitly?

Reply:
Yeah, we can make it explicit. I think I am only hoping this argument not showing up in the arg list of `init_model_parallel`.

I think it's good now.

Reply:
We can move `num_send_recv_threads` to `backend_config` once we have updated the `init_model_parallel` API. 

+1 it's good for now

Reply:
@mrshenli 
No worry here. `num_send_recv_threads ` will be put into the `RpcAgentOptions` in my coming PR.